bunk.md

```diff
@@ -0,0 +1,2 @@
+Omar Little: Shoot, the way y'all looking at things, ain't no victim to even speak on.
+Det. William 'Bunk' Moreland: Bullshit, boy. No victim? I just came from Tosha's people, remember? All this death, you don't think it ripples out? You don't even know what the fuck I'm talking about. I was a few years ahead of you at Edmondson, but I know you remember the neighborhood, how it was. We had some bad boys, for real. Wasn't about guns so much as knowing what to do with your hands. Those boys could really rack. My father had me on the straight, but like any young man, I wanted to be hard too, so I'd turn up at all the house parties where the tough boys hung. Shit, they knew I wasn't one of them. Them hard cases would come up to me and say, "Go home, schoolboy, you don't belong here." Didn't realize at the time what they were doing for me. As rough as that neighborhood could be, we had us a community. Nobody, no victim, who didn't matter. And now all we got is bodies, and predatory motherfuckers like you. And out where that girl fell, I saw kids acting like Omar, calling you by name, glorifying your ass. Makes me sick, motherfucker, how far we done fell.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.